Scripts/Mobiles/Normal/GiantSpider.cs

```diff
@@ -10,39 +10,39 @@ public class GiantSpider : BaseCreature
         public GiantSpider()
             : base(AIType.AI_Melee, FightMode.Closest, 10, 1, 0.2, 0.4)
         {
-            this.Name = "a giant spider";
-            this.Body = 28;
-            this.BaseSoundID = 0x388;
+            Name = "a giant spider";
+            Body = 28;
+            BaseSoundID = 0x388;
 
-            this.SetStr(76, 100);
-            this.SetDex(76, 95);
-            this.SetInt(36, 60);
+            SetStr(76, 100);
+            SetDex(76, 95);
+            SetInt(36, 60);
 
-            this.SetHits(46, 60);
-            this.SetMana(0);
+            SetHits(46, 60);
+            SetMana(0);
 
-            this.SetDamage(5, 13);
+            SetDamage(5, 13);
 
-            this.SetDamageType(ResistanceType.Physical, 100);
+            SetDamageType(ResistanceType.Physical, 100);
 
-            this.SetResistance(ResistanceType.Physical, 15, 20);
-            this.SetResistance(ResistanceType.Poison, 25, 35);
+            SetResistance(ResistanceType.Physical, 15, 20);
+            SetResistance(ResistanceType.Poison, 25, 35);
 
-            this.SetSkill(SkillName.Poisoning, 60.1, 80.0);
-            this.SetSkill(SkillName.MagicResist, 25.1, 40.0);
-            this.SetSkill(SkillName.Tactics, 35.1, 50.0);
-            this.SetSkill(SkillName.Wrestling, 50.1, 65.0);
+            SetSkill(SkillName.Poisoning, 60.1, 80.0);
+            SetSkill(SkillName.MagicResist, 25.1, 40.0);
+            SetSkill(SkillName.Tactics, 35.1, 50.0);
+            SetSkill(SkillName.Wrestling, 50.1, 65.0);
 
-            this.Fame = 600;
-            this.Karma = -600;
+            Fame = 600;
+            Karma = -600;
 
-            this.VirtualArmor = 16;
+            VirtualArmor = 16;
 
-            this.Tamable = true;
-            this.ControlSlots = 1;
-            this.MinTameSkill = 59.1;
+            Tamable = true;
+            ControlSlots = 1;
+            MinTameSkill = 59.1;
 
```

Comment:
missing SetMagicalAbility(MagicalAbility.Poisoning);

Reply:
No, its set in basecreature. Adding it during versioning was for pre-existing creatures.